docs: add percentage type support for node.search.cache.size (#12892)
opensearch-project/documentation-website: _tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot.md

Parameter | Type | Description
:--- | :--- | :---

> `node.search.cache.size` | String | Specifies the cache size either as an absolute byte value (for example, `7kb` or `6gb`, using the units described in [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/)) or as a percentage of the total disk space (for example, `10%`).

Reviewer: ```suggestion
`node.search.cache.size` | String | Specifies the cache size as either an absolute byte size (for example, `7kb` or `6gb`) or a percentage of the total disk space (for example, `10%`). For more information about byte size units, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
```
